cli/internal/oauth/api/api.go

```diff
@@ -96,17 +96,31 @@ func tryDecodeOAuthError(resp *http.Response) error {
 // authenticated or we have reached the time limit for authenticating (based on
 // the response from GetDeviceCode).
 func (a API) WaitForDeviceToken(ctx context.Context, state State) (TokenResponse, error) {
-	ticker := time.NewTicker(state.IntervalDuration())
+	// Ticker for polling tenant for login – based on the interval
+	// specified by the tenant response.
+	ticker := time.NewTimer(state.IntervalDuration())
 	defer ticker.Stop()
-	timeout := time.After(state.ExpiryDuration())
+	// The tenant tells us for as long as we can poll it for credentials
+	// while the user logs in in the browser. Timeout if we don't get
```

Comment:
Heh; linter doesn't like the comment;

```
68.69 cli/internal/oauth/api/api.go:104:2: Duplicate words (in) found (dupword)
68.69 	// while the user logs in in the browser. Timeout if we don't get
68.69 	^
```

Perhaps "logs in through the browser" or "logs in using the browser" ?